packages/static-build/src/index.ts

```diff
@@ -707,7 +707,7 @@ export const build: BuildV2 = async ({
           routes.push(...frameworkRoutes);
         }
 
-        let ignore: string[] = [];
+        let ignore: string[] = ['.vercel/**'];
         if (config.zeroConfig && config.outputDirectory === '.') {
```

Comment:
There should be a second case with `if (!config.zeroConfig && config.distDir === '.')`

Reply:
Or we could use a single 

`if (config.outputDirectory === '.' || config.distDir === '.')`